application/org.openjdk.jmc.flightrecorder.flameview/src/main/java/org/openjdk/jmc/flightrecorder/flameview/tree/TraceTreeUtils.java

```diff
@@ -43,6 +43,7 @@
 import org.openjdk.jmc.flightrecorder.stacktrace.StacktraceModel.Fork;
 import org.openjdk.jmc.flightrecorder.stacktrace.FrameSeparator.FrameCategorization;
 import org.openjdk.jmc.flightrecorder.stacktrace.StacktraceFrame;
+import org.openjdk.jmc.flightrecorder.stacktrace.messages.internal.Messages;
```

Comment:
Messages to be shared should not be in an internal bundle.

Reply:
I've put them into the move them into the package : org.openjdk.jmc.flightrecorder.stacktrace.messages.common.

Reply:
moved to the org.openjdk.jmc.flightrecorder.stacktrace